pages/nation.md

```diff
@@ -34,10 +34,10 @@ Then, you should see a list of communities and the option to generate a report i
 
 This is the usual sync process from the community side. There are three other important kinds of updates that you receive on the community side: updates, publications, and surveys.  
 
-As you can see from the image below, there is an update, a publication, and a survey ready for download. 
+As you can see from the image below, there is an update, a publication, and a survey ready for download. Don't worry if you don't see the release notes button, the image below is just to illustrate all buttons you might possibly see.
 
 ![Update, Publication, and Survey ready from the nation](uploads/images/nation.md7.png)
 
-First, click the "Update Available" button and it will reload your homepage with a successful update message. An update refers to a new software update which improves the software. Next, click on "Publications" and sync the publication. Publications add new resources or courses to your library. 
+First, click the "Update Available" button and it will reload your homepage with a successful update message. An update refers to a new software update which improves the software. Next, click on "Publications" and sync the publication.
 
```

Comment:
That sentence doesn't need deleting. It is useful to know what publications are, isn't it?
Instead, add here the last words of line 43 ("repeat the process of sending an activity sync to the nation.").